### PR TITLE
Storage get count Review

### PR DESCRIPTION
**Pull request by Ineffable22**
Verify that this code meets the requirements set for the corresponding task.

### CODE REVIEW

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -74,3 +74,18 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """returns the object based on the class and its ID, or None if not found"""
```
Comment (humtej1204) [inline models/engine/db_storage.py]: Please double-check the pycodestyle, it has 5 characters more than allowed.

File: models/engine/file_storage.py [hunk 2]
```diff
@@ -68,3 +69,17 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """returns the object based on the class and its ID, or None if not found"""
```
Comment (humtej1204) [inline models/engine/file_storage.py]: Please double-check your pycodestyle, you have 5 characters more than allowed

**Review by humtej1204 — COMMENTED** [review]
Please check the revisions made

File: models/engine/db_storage.py (hunk 1 above)
Comment (Ineffable22) [reply models/engine/db_storage.py]: ok it was solved

File: models/engine/file_storage.py (hunk 2 above)
Comment (Ineffable22) [reply models/engine/file_storage.py]: ok it was solved

**Review by humtej1204 — APPROVED** [review]
All changes are ok

**Pull request merged**